netcore/runtime.nuspec

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>runtime.$RID$.Microsoft.NETCore.Runtime.Mono</id>
+        <version>$VERSION$</version>
+        <description>Internal implementation package not meant for direct consumption.  Please do not reference directly. 
+The Mono runtime, and the base library, called mscorlib. It includes the garbage collector, JIT compiler, base .NET data types and many low-level classes.</description>
```

Comment:
Nit: There's no mscorlib, only System.Private.CoreLib

Reply:
I copy-pasted the description from the .NET Core runtime nupkg, which also has no mscorlib :laughing:

Reply:
Oh yeah. It had it by the time it was written, someone just forgot to update it :)